src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js

```diff
@@ -0,0 +1,102 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
+import { PROFILE_PATHS } from '@@profile/constants';
+
+import mockUserInEVSS from '@@profile/tests/fixtures/users/user-36.json';
+import mockDD4CNPNotEnrolled from '@@profile/tests/fixtures/dd4cnp/dd4cnp-is-not-set-up.json';
+import mockDD4CNPEnrolled from '@@profile/tests/fixtures/dd4cnp/dd4cnp-is-set-up.json';
+import mockDD4EDUEnrolled from '@@profile/tests/fixtures/dd4edu/dd4edu-enrolled.json';
+
+// TODO: remove this when we are no longer gating DD4EDU with a feature flag
+const dd4eduEnabled = {
+  data: {
+    type: 'feature_toggles',
+    features: [
+      {
+        name: 'ch33_dd_profile',
+        value: true,
+      },
+    ],
+  },
+};
+
+function fillInBankInfoForm() {
```

Comment:
Made a bunch of helpers simply because it makes the actual test easier to read, IMO.